addons/binding/org.openhab.binding.allplay/OSGI-INF/.gitignore

```diff
@@ -0,0 +1 @@
+/*.xml
```

Comment:
Do we really need and more important want all these gitgnores on all levels, if its true for OSGI-INF files could we maybe use:

`OSGI-INF/*.xml`